src/script/cpp_api/s_base.cpp

```diff
@@ -160,18 +160,18 @@ int ScriptApiBase::luaPanic(lua_State *L)
 void ScriptApiBase::clientOpenLibs(lua_State *L)
 {
 	static const std::vector<std::pair<std::string, lua_CFunction>> m_libs = {
-		{ "", luaopen_base },
-		{ LUA_TABLIBNAME,  luaopen_table   },
-		{ LUA_OSLIBNAME,   luaopen_os      },
-		{ LUA_STRLIBNAME,  luaopen_string  },
-		{ LUA_MATHLIBNAME, luaopen_math    },
-		{ LUA_DBLIBNAME,   luaopen_debug   },
+		{"", luaopen_base},
+		{LUA_TABLIBNAME, luaopen_table},
+		{LUA_OSLIBNAME, luaopen_os},
+		{LUA_STRLIBNAME, luaopen_string},
+		{LUA_MATHLIBNAME, luaopen_math},
+		{LUA_DBLIBNAME, luaopen_debug},
 #if USE_LUAJIT
-		{ LUA_JITLIBNAME,  luaopen_jit     },
+		{LUA_JITLIBNAME, luaopen_jit},
```

Comment:
the entries were all aligned before

Reply:
clang-format file format settings were redefined and did not allow indentation

Reply:
fix it https://github.com/minetest/minetest/pull/12704/commits/50efdb084bf3c4a1af5757106c26a1ccbf34134d